Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/shopify/use-category.md

# useCategory composable

> `useCategory` composition API function is responsible, as its name suggests, for interactions with categories from your eCommerce. This function returns the following values:

Reviewer: ```suggestion
`useCategory` composition API function allows loading the categories. This function returns the following values:
```
